src/components/NavBar/OidcLoginComponent.jsx

```diff
@@ -33,8 +33,7 @@ const OidcLoginComponent = () => {
           }
         }}
         InputProps={{
-          disableUnderline: true,
-          'aria-label': 'OIDC Input Field'
```

Comment:
Differs from `label` prop, which should be used instead

Reply:
you said `label` prop should be used?

Reply:
Yeah, the [`label` that is a top-level prop](https://mui.com/material-ui/api/text-field/#TextField-prop-label) (in this case, "Pod Server URL"), not an object property passed into [`InputProps`](https://mui.com/material-ui/api/text-field/#TextField-prop-InputProps).